pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions_test.go

```diff
@@ -52,26 +53,33 @@ func TestGCEDiskConflicts(t *testing.T) {
 	}
 	errStatus := framework.NewStatus(framework.Unschedulable, ErrReasonDiskConflict)
 	tests := []struct {
-		pod        *v1.Pod
-		nodeInfo   *framework.NodeInfo
-		isOk       bool
-		name       string
-		wantStatus *framework.Status
+		pod                 *v1.Pod
+		nodeInfo            *framework.NodeInfo
+		isOk                bool
+		name                string
+		wantFilterStatus    *framework.Status
+		wantPreFilterStatus *framework.Status
 	}{
-		{&v1.Pod{}, framework.NewNodeInfo(), true, "nothing", nil},
-		{&v1.Pod{}, framework.NewNodeInfo(st.MakePod().Volume(volState).Obj()), true, "one state", nil},
-		{st.MakePod().Volume(volState).Obj(), framework.NewNodeInfo(st.MakePod().Volume(volState).Obj()), false, "same state", errStatus},
-		{st.MakePod().Volume(volState2).Obj(), framework.NewNodeInfo(st.MakePod().Volume(volState).Obj()), true, "different state", nil},
+		{&v1.Pod{}, framework.NewNodeInfo(), true, "nothing", nil, framework.NewStatus(framework.Skip)},
+		{&v1.Pod{}, framework.NewNodeInfo(st.MakePod().Volume(volState).Obj()), true, "one state", nil, framework.NewStatus(framework.Skip)},
+		{st.MakePod().Volume(volState).Obj(), framework.NewNodeInfo(st.MakePod().Volume(volState).Obj()), false, "same state", errStatus, nil},
+		{st.MakePod().Volume(volState2).Obj(), framework.NewNodeInfo(st.MakePod().Volume(volState).Obj()), true, "different state", nil, nil},
 	}
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 			p := newPlugin(ctx, t)
-			gotStatus := p.(framework.FilterPlugin).Filter(context.Background(), nil, test.pod, test.nodeInfo)
-			if !reflect.DeepEqual(gotStatus, test.wantStatus) {
-				t.Errorf("status does not match: %v, want: %v", gotStatus, test.wantStatus)
+			_, preFilterStatus := p.(framework.PreFilterPlugin).PreFilter(ctx, nil, test.pod)
+			if diff := cmp.Diff(preFilterStatus, test.wantPreFilterStatus); diff != "" {
+				t.Errorf("status does not match (-want,+got):\n%s", diff)
```

Comment:
Ditto for other tests in this file

```suggestion
				t.Errorf("PreFilter: status does not match (-want,+got):\n%s", diff)
```